_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -1,1420 +1,1661 @@
+/obj/effect/turf_decal/industrial/warning{
+	dir = 8
+	},
+/obj/effect/turf_decal/industrial/warning{
 	dir = 4
 	},
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
-	dir = 10
+/obj/docking_port/mobile{
+	dir = 4;
+	launch_status = 0;
+	preferred_direction = 4;
+	port_direction = 2
+	},
+/obj/machinery/door/poddoor/shutters{
+	dir = 8;
+	id = "traumalobby";
+	name = "Lobby"
 	},
+/turf/open/floor/engine,
+/area/ship/cargo)
+"sI" = (
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
 /obj/structure/cable{
 	icon_state = "1-2"
 	},
-/obj/effect/turf_decal/corner/opaque/white/mono,
-/turf/open/floor/plasteel/white,
+/obj/machinery/light/small/directional/east,
+/turf/open/floor/plating,
+/area/ship/maintenance/starboard)
+"sO" = (
+/obj/effect/turf_decal/industrial/outline/red,
+/obj/structure/bodycontainer/morgue{
+	dir = 1
+	},
+/obj/machinery/light/directional/south,
+/turf/open/floor/plasteel/patterned/brushed,
 /area/ship/medical)
-"su" = (
-/obj/effect/landmark/start/paramedic,
-/turf/open/floor/plasteel,
-/area/ship/crew)
-"sz" = (
-/obj/effect/turf_decal/trimline/opaque/red/filled/line{
+"tb" = (
+/obj/effect/turf_decal/industrial/hatch/yellow,
+/obj/structure/reagent_dispensers/watertank,
+/obj/item/mop,
+/obj/item/reagent_containers/glass/bucket,
+/obj/item/storage/bag/trash,
+/obj/machinery/light/small/directional/east,
+/turf/open/floor/plating,
+/area/ship/maintenance/starboard)
+"tF" = (
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/effect/turf_decal/spline/fancy/opaque/black{
 	dir = 4
 	},
-/obj/machinery/door/firedoor/border_only,
-/obj/machinery/door/firedoor/border_only{
-	dir = 1
+/obj/machinery/atmospherics/components/unary/vent_pump/on/layer2,
+/obj/effect/turf_decal/spline/fancy/opaque/black{
+	dir = 8
 	},
-/obj/item/radio/intercom/wideband/directional/east,
-/turf/open/floor/plasteel/white,
-/area/ship/hallway/fore)
-"sH" = (
-/obj/machinery/atmospherics/pipe/manifold/scrubbers/hidden/layer4,
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+/turf/open/floor/plasteel/patterned/brushed,
+/area/ship/medical)
+"tI" = (
+/obj/machinery/atmospherics/pipe/layer_manifold{
 	dir = 4
 	},
-/obj/structure/cable{
-	icon_state = "4-8"
+/obj/machinery/door/poddoor/preopen{
+	dir = 8;
+	id = "traumawindows";
+	name = "Window Blast Door"
 	},
+/obj/structure/grille,
+/obj/structure/window/reinforced/fulltile/shuttle,
 /turf/open/floor/plating,
 /area/ship/maintenance/port)
-"sM" = (
-/obj/machinery/atmospherics/pipe/manifold4w/scrubbers/hidden/layer4,
-/obj/machinery/atmospherics/pipe/manifold4w/supply/hidden/layer2,
-/obj/structure/cable{
-	icon_state = "2-4"
+"tJ" = (
+/obj/structure/sign/poster/official/cleanliness{
+	pixel_x = 32
 	},
-/obj/structure/cable{
-	icon_state = "2-8"
+/obj/effect/turf_decal/industrial/outline/red,
+/obj/structure/closet/crate/freezer/surplus_limbs/organs,
```

Comment:
Replace this crate with a freezer crate containing basic cybernetic organs instead of biological ones.